deploy/scripts/000020-set-network-settings.ts

```diff
@@ -34,13 +34,6 @@ const func: DeployFunction = async ({ getNamedAccounts }: HardhatRuntimeEnvironm
         from: deployer
     });
 
-    await execute({
-        name: InstanceName.NetworkSettings,
-        methodName: 'setVortexRewards',
-        args: [{ burnRewardPPM: toPPM(10), burnRewardMaxAmount: toWei(100) }],
-        from: deployer
-    });
-
```

Comment:
TODO:
1. Reinstate
2. Rename to V1
3. Add a deployment script for the new version
4. Add a deployment test for the new version
5. Add a deployment script for the BancorVortex contract
6. Add a deployment test for the BancorVortex contract
7. Import the BancorVortex artifact in `/utils/Deploy.ts`